privileges-grantor/README.md

```diff
@@ -11,8 +11,7 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | instance\_name | Privileges Grantor instance name to identify resources in multi-instance deployments. | string | `""` | no |
 | lambda\_timeout | The time after which the lambda execution stops. | string | `"200"` | no |
 | memory | The amount of memory (in MiB) to be used by Lambda | string | `"512"` | no |
-| metastore\_events\_filter | List of metastore event types to be added to SNS filter. Supported format: `[ "CREATE_TABLE","ALTER_TABLE" ]` | list | `[ "CREATE_TABLE", "ALTER_TABLE" ]` | no |
-| database\_filter | List of database names to be added to SNS filter. Supported format: `[ "DB_NAME_1", "DB_NAME_2" ]` | list | n/a | yes |
+| metastore\_events\_sns\_filter | SNS filter of message attributes to the added to the sns sns topic subscription. Supported format: SNS filter format | string | `{ "eventType": [ "CREATE_TABLE", "ALTER_TABLE" ] }` | no |
```

Comment:
So have we changed the name of this variable? If so this is a backwards incompatible change for end users and we need to document that clearly and do a major version bump. Or was this just wrong before and we've fixed it?

Reply:
we dropped 2 old input vars and replaced them with new one that will hold the SNS subscription filter in JSON. This is because it is very difficult to construct a proper JSON with random complexity out of terraform variables in terraform 0.11. Terraform treats curly and square brackets as special characters (as internally it represents lists and maps as strings) and thus you can't really use terraform's interpolation syntax to produce a valid JSON and wrap it in the end with jsonencode()

Reply:
Sounds fun! Thanks for the clarification. We should all go back to using XML :)